javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,7 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  for (var i = values.length - 1; i < values.length; i++) {
+    var filter = values.filter(Element => typeof Element !== 'string');
```

Comment:
Make sure to change the first letter to lowercase. You want most variables to start with a lowercase letter. Uppercase is for specific types of variables.